flipperdevices/flipperzero-firmware
# FL-819 new first start screen, add multiline text

# What's new

- Dolphin: smaller first start screen assets, updated to latest version
- Gui: new multiline rendering element

# Verification

- Build and upload
- See new start screens

# Checklist (do not modify)

- [x] Cross-validation by @glitchcore
